sample-apps/js-sdk-aws-XRay/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sample-app",
+  "version": "0.11.0",
+  "description": "Sample Instrumentation of X-Ray",
+  "main": "index.js",
+  "scripts": {
+    "zipkin:server": "cross-env EXPORTER=zipkin node ./server.js",
+    "zipkin:client": "cross-env EXPORTER=zipkin node ./client.js",
+    "jaeger:server": "cross-env EXPORTER=jaeger node ./server.js",
+    "jaeger:client": "cross-env EXPORTER=jaeger node ./client.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "http",
+    "tracing"
+  ],
+  "engines": {
+    "node": ">=8"
+  },
+  "dependencies": {
```

Comment:
you will need to build this image with the repo: https://github.com/aws-observability/aws-otel-js whenever there's code commit, so i'm thinking how do we put the dependencies here, looks like it's using the official repo instead? 

maybe try to add aws-itel-js as a local lib? https://stackoverflow.com/questions/48944051/package-json-add-local-directory-to-node-modules/48944131

Reply:
Will be moving to aws-otel-js repository to resolve dependencies issues